lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -71,6 +71,8 @@ class SPIRVModuleImpl : public SPIRVModule {
     MemoryModel(MemoryModelOpenCL){

```

Comment:
addCapability should still be called since setMemoryModel will set the same value as this line.
I agree with Sam - line #71 is redundant.